### PR TITLE
Protobuf sbt plugin for sbt 1.0

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/Protobuf.scala [hunk 1]
```diff
@@ -1,87 +1,131 @@
...
+    IO.reader(source) { reader =>
+      IO.writer(target, "", IO.defaultCharset) { writer =>
+        val pw = new PrintWriter(writer)
+        IO.foreachLine(reader) { line => pw.println(transform(line)) }
+      }
+    }
+  }
+
+
+}
```
Comment (ktoso) [inline project/Protobuf.scala]: Oh so extracted the feature we used here from sbt-site?

**Review by ktoso — APPROVED** [review]
Nice, LGTM :)

**Comment by ktoso** [comment]
(merge at will, since into the WIP-build branch anyway)

File: project/Protobuf.scala (hunk 1 above)
Comment (raboof) [reply project/Protobuf.scala]: Yeah, it became private there, so I took the 'easy' way out and extracted it.

**Review by patriknw — APPROVED** [review]
LGTM

File: project/Protobuf.scala (hunk 1 above)
Comment (ktoso) [reply project/Protobuf.scala]: 👍  good; power of open source ;-)

**Comment by ktoso** [comment]
Failure was formatting, which we ignore on this branch for now (since no formatted plugin)

**Pull request merged**